frame/support/src/storage/types/counted_map.rs

```diff
@@ -543,6 +543,16 @@ mod test {
 			97
 		}
 	}
+	#[crate::storage_alias]
+	type ExampleCountedMap = CountedStorageMap<Prefix, Twox64Concat, u16, u32>;
+
+	#[test]
+	fn storage_alias_works() {
+		TestExternalities::default().execute_with(|| {
+			assert_eq!(ExampleCountedMap::count(), 0);
+			ExampleCountedMap::insert(3, 10);
```

Comment:
Could we add some test that the storage alias works the same as the normal counted storage map from the pallet macro?

Reply:
Basically insert and count are the two functions that prove that CountedStorageMap works as intended thru the macro, otherwise it does not compile. 

The rest of the functionality is tested for the normal CountedStorageMap.

Reply:
But I guess testing the prefixes would be nice, will do!

Reply:
Added some tests alongside pallet storage, which revealed a couple of issues. Seems to be all good now!